client/cmd/status.go

```diff
@@ -639,6 +659,11 @@ func parsePeers(peers peersStateOutput, rosenpassEnabled, rosenpassPermissive bo
 				"  Transfer status (received/sent) %s/%s\n"+
 				"  Quantum resistance: %s\n"+
 				"  Routes: %s\n",
+				"  Transfer status (received/sent): %s/%s\n"+
```

Comment:
Is this a dupe?

Reply:
Ah yeah I think there went something wrong during merge. I will check the default status output